src/main/java/org/openelisglobal/testconfiguration/controller/TestModifyEntryController.java

```diff
@@ -542,8 +546,10 @@ private List<TestSet> createTestSets(TestAddParams testAddParams) {
             highValid = StringUtil.doubleWithInfinity(testAddParams.highValid);
             lowReportingRange = StringUtil.doubleWithInfinity(testAddParams.lowReportingRange);
             highReportingRange = StringUtil.doubleWithInfinity(testAddParams.highReportingRange);
-            lowCriticalRange = StringUtil.doubleWithInfinity(testAddParams.lowCriticalRange);
-            highCriticalRange = StringUtil.doubleWithInfinity(testAddParams.highCriticalRange);
+            lowRangeLow = StringUtil.doubleWithInfinity(testAddParams.lowRangeLow);
```

Comment:
Same note as above, omit passing the doubles into createResultLimits and move the conversion of StringUtil.doubleWithInfinity(value) into the method